src/script/telemetry/app_init/AppInitTimings.ts

```diff
@@ -51,7 +51,7 @@ export class AppInitTimings {
     const appLoaded = this.timings[AppInitTimingsStep.APP_LOADED];
     const appLoadedInSeconds = appLoaded / TIME_IN_MILLIS.SECOND;
 
-    return (Math.floor(appLoadedInSeconds / AppInitTimings.CONFIG.BUCKET_SIZE) + 1) * AppInitTimings.CONFIG.BUCKET_SIZE;
```

Comment:
I'm a bit curious about what was the big idea there

Reply:
The big idea was to over-engineer a piece of code that has very little value (it always say that the app loaded in less than 10s, we could just print the actual number of seconds)